lib/src/serializers.dart

```diff
@@ -324,7 +324,16 @@ Serializers serializers = _$serializers;
 Serializers standard_serializers = (serializers.toBuilder()
       ..add(PointSerializer<num>())
       ..add(ColorSerializer())
-      ..addPlugin(new StandardJsonPlugin()))
+      ..addPlugin(new StandardJsonPlugin())
```

Comment:
Can you add a link to a webpage explaining this? (assuming there is one that suggested this fix)